### DISCUSSION TITLE
thenounproject vs heroicons

### DISCUSSION THREAD

**Discussion by anjanesh** [tailwindlabs/tailwindcss#9259] [Help]
I have an account at thenounproject.com - I want to use some icons from there to use in examples provided in [sidebar](https://tailwindui.com/components/application-ui/application-shells/sidebar).

The heroicons show crisp and clear - but when I replace heroicons with the thenounproject icons of same size the clarity is low and not as dark as heroicons.

How do I get the thenounproject icons to show as crisp as the heroicons on a 24x24 size image ?

**Comment by wongjn** [comment]
To ensure maximum crispness, you should ensure that you are using SVGs from thenounproject.com icons, and also that they have been drawn to a canvas size is a multiple of the 24×24 size, such as 24×24, 48×48, etc. 32×32 for example would not look as crisp.
